modules/tinymce/src/themes/silver/test/ts/browser/editor/contextmenu/DesktopContextMenuTest.ts

```diff
@@ -120,36 +112,6 @@ describe('browser.tinymce.themes.silver.editor.contextmenu.DesktopContextMenuTes
     await pWaitForAndCloseDialog(editor);
   });
 
-  it('TBA: Test context menus on image inside a table', async () => {
```

Comment:
Likewise this shouldn't have been removed.

- [ ] Revert and only remove imagetools specific logic